winter/http/throttling.py

```diff
@@ -88,16 +88,16 @@ def create_throttle_classes(
         routes: typing.List['Route'],
 ) -> typing.Tuple[typing.Type[BaseRateThrottle], ...]:
     base_throttling_annotation = component.annotations.get_one_or_none(ThrottlingAnnotation)
-    throttling_by_http_method_: typing.Dict[str, typing.Optional[Throttling]] = {}
+    throttling_by_http_method_: typing.Dict[str, Throttling] = {}
 
     for route in routes:
 
-        throttling_annotation = route.method.annotations.get_one_or_none(ThrottlingAnnotation)
+        throttling_annotation = (
+            route.method.annotations.get_one_or_none(ThrottlingAnnotation) or
```

Comment:
Consider `route.method.annotations.get_one_or_default(ThrottlingAnnotation, base_throttling_annotation)`

Reply:
I will do it in another PR:
What do you think about only one method:
get_one(Route) -> get_one(Route) 
get_one_or_none(Route) -> get_one(Route, None)
get_one_or_default(Route, default) -> get_one(Route, default)